app/controllers/deliveries/show/resend.js

```diff
@@ -1,22 +1,36 @@
 import Ember from 'ember';
+import CanResendController from './can-resend-controller';
 
-export default Ember.Controller.extend({
+export default CanResendController.extend({
+  errors: [],
+  errorMessages: Ember.computed('errors', function () {
```

Comment:
FYI, an [Ember.computed.mapBy](https://emberjs.com/api/ember/2.7/classes/Ember.computed/methods?anchor=mapBy) can do this in one line :smile: